FireRender.Maya.Src/FireRenderProduction.cpp

```diff
@@ -249,163 +255,180 @@ bool FireRenderProduction::start()
 	// Read common render settings.
 	MRenderUtil::getCommonRenderSettings(m_settings);
 
-	// Read RPR globals.
-	m_globals.readFromCurrentScene();
-
-	// Disable tile rendering for RPR2
-
 	MString renderStamp;
 	if (m_globals.useRenderStamp)
 		renderStamp = m_globals.renderStampText;
 
 	m_aovs = &m_globals.aovs;
 
-	int contextWidth = m_width;
-	int contextHeight = m_height;
+	{
+		AutoMutexLock contextCreationLock(m_contextCreationLock);
```

Comment:
why we need this mutex on context creation phase ?

Reply:
this is from legacy code